Added pull request template with code and content change checklist (#156)
HO-CTO/engineering-guidance-and-standards: .github/PULL_REQUEST_TEMPLATE.md

Is this pull request a content or a code change?

# Code change
I can confirm:

> - [ ] Manual accessibility testing has been performed (if appropriate)

Reviewer: ```suggestion
- [ ] This change will not change layouts, page structures or anything else that might impact accessibility
or
- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
(If the change might impact accessibility then please add some further information here)
```

# Content change 
I can confirm:

> - [ ] Content is suitable to open source
> - [ ] Content meets the content standards
> - [ ] Content does not relate to unreleased gov policy
> - [ ] Content is appropriately tagged
> - [ ] Content does not refer to anti-fraud mechanisms
> - [ ] Content does not include any code or configuration changes (excluding frontmatter information)
> - [ ] Content does not include sensitive business logic

Reviewer: Can scratch 'content is appropriately tagged' - it is part of the content standards

```suggestion
- [ ] Content does not include any code or configuration changes (excluding frontmatter information)
- [ ] Content meets the content standards
e.g. [Writing a principle](https://ho-cto.github.io/engineering-guidance-and-standards/docs/standards/writing-a-principle/) and [Writing a standard](https://ho-cto.github.io/engineering-guidance-and-standards/docs/standards/writing-a-standard/)
- [ ] Content is suitable to open source
i.e.
- Content does not relate to unreleased gov policy
- Content does not refer to anti-fraud mechanisms
- Content does not include sensitive business logic
```

Is this pull request a content or a code change?

# Code change
I can confirm:

> - [ ] This change will not change layouts, page structures or anything else that might impact accessibility
> or
> - [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
> (If the change might impact accessibility then please add some further information here)

Reviewer: Sorry, just rethought this
```suggestion
## Accessibility considerations
- [ ] This change will not change layouts, page structures or anything else that might impact accessibility
or
- [ ] This change might impact accessibility, automated aXe tests cover the impact
or
- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
(If the change might impact accessibility then please add some further information here)
```

> Is this pull request a content or a code change?

Reviewer 2: ```suggestion
Is this pull request a content or a code change? (Please fill in the relevant section and delete the other)
```

# Code change
I can confirm:
## Accessibility considerations

> - [ ] This change will not change layouts, page structures or anything else that might impact accessibility
> or
> - [ ] This change might impact accessibility, automated aXe tests cover the impact
> or
> - [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
> (If the change might impact accessibility then please add some further information here)

Reviewer 2: When rendered in GH the `or` is appended to the previous line, so isn't obvious. Add line breaks to have them render as a new paragraph.
```suggestion
- [ ] This change will not change layouts, page structures or anything else that might impact accessibility

or
- [ ] This change might impact accessibility, automated aXe tests cover the impact

or
- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
(If the change might impact accessibility then please add some further information here)
```

# Content change 
I can confirm:
- [ ] Content does not include any code or configuration changes (excluding frontmatter information)
- [ ] Content meets the content standards
e.g. [Writing a principle](https://ho-cto.github.io/engineering-guidance-and-standards/docs/standards/writing-a-principle/) and [Writing a standard](https://ho-cto.github.io/engineering-guidance-and-standards/docs/standards/writing-a-standard/)

> - [ ] Content is suitable to open source
> i.e.
> - Content does not relate to unreleased gov policy
> - Content does not refer to anti-fraud mechanisms
> - Content does not include sensitive business logic

Reviewer 2: ```suggestion
- [ ] Content is suitable to open source, i.e.:
    - Content does not relate to unreleased gov policy
    - Content does not refer to anti-fraud mechanisms
    - Content does not include sensitive business logic
```
